src/dmd/dinterpret.d

```diff
@@ -2595,12 +2595,21 @@ public:
         {
             printf("%s StructLiteralExp::interpret() %s ownedByCtfe = %d\n", e.loc.toChars(), e.toChars(), e.ownedByCtfe);
         }
-        if (e.ownedByCtfe >= OwnedBy.ctfe)
+        if (e.ownedByCtfe == OwnedBy.ctfe ||
+            e.ownedByCtfe == OwnedBy.cache)
         {
             result = e;
             return;
         }
 
+        if (e.stageflags & stageScrub)
+        {
+            result = e;
+            return;
+        }
+        const oldflags = e.stageflags;
+//        e.stageflags |= stageScrub;
```

Comment:
Don't comment out code here Walter. That is the standard of @atilaneves has been requested of us.